lib/cqm/converter/hds_record.rb

```diff
@@ -72,7 +72,11 @@ def to_qdm(record)
       ethnicity = record.ethnicity
       if ethnicity
         # See: https://phinvads.cdc.gov/vads/ViewCodeSystem.action?id=2.16.840.1.113883.6.238
-        code = QDM::Code.new(ethnicity['code'], 'cdcrec', ethnicity['name'], '2.16.840.1.113883.6.238')
+        # Bonnie currently uses 'CDC Race' instead of the correct 'cdcrec'.  This duplicate code is here as a temporary
+        # workaround until the larger change of making bonnie use 'cdcrec' can be implemented.
+        # Same change is present in `race` below.
+        code = QDM::Code.new(ethnicity['code'], 'CDC Race', ethnicity['name'], '2.16.840.1.113883.6.238')
+        # code = QDM::Code.new(ethnicity['code'], 'cdcrec', ethnicity['name'], '2.16.840.1.113883.6.238')
```

Comment:
It doesn't look like this is adding the duplicate code. It looks like the single wrong code is being added.

Reply:
Ah good catch, stale comment.  Will update